includes/options/class-amp-options-manager.php

```diff
@@ -41,15 +51,59 @@ public static function register_settings() {
 			self::OPTION_NAME,
 			self::OPTION_NAME,
 			[
-				'type'              => 'array',
+				'type'              => 'object',
 				'sanitize_callback' => [ __CLASS__, 'validate_options' ],
+				'show_in_rest'      => [
```

Comment:
attn: @pierlon I removed the REST endpoint for options but had to define the schema here. Saved a lot of code, but the drawback is that some of these options are not needed for the immediate purposes of the wizard app. I'm able to handle that on the React side, though, by only updating the relevant properties.

Reply:
Sounds good.